docs/modules/hive/pages/usage-guide/database-driver-example.adoc

```diff
@@ -0,0 +1,176 @@
+              "-x",
+              "-c",
+              "mkdir -p ${DEST_DIR} && chown stackable:stackable ${DEST_DIR} && chmod -R a=,u=rwX,g=rwX ${DEST_DIR}",
+            ]
+          securityContext:
+            runAsUser: 0
+          volumeMounts:
+            - name: external-drivers
+              mountPath: /stackable/externals
+      containers:
+        - name: hive-driver
+          image: docker.stackable.tech/stackable/tools:1.0.0-stackable24.3.0
+          env:
+            - name: DEST_DIR
+              value: "/stackable/externals"
+          command:
+            [
+              "bash",
+              "-x",
+              "-c",
+              "curl -L  https://repo1.maven.org/maven2/com/mysql/mysql-connector-j/8.0.31/mysql-connector-j-8.0.31.jar \
+              -o ${DEST_DIR}/mysql-connector-j-8.0.31.jar",
+            ]
+          volumeMounts:
+            - name: external-drivers
+              mountPath: /stackable/externals
+----
+
+This will make the driver available at `/stackable/external-drivers/mysql-connector-j-8.0.31.jar` when the volume `external-drivers` is mounted at `/stackable/external-drivers`.
+
+Once the above has completed successfully, we can confirm that the driver is in the expected location by running another job:
+
+[source,yaml]
+----
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: list-drivers-job
+spec:
+  template:
+    spec:
+      restartPolicy: Never
+      volumes:
+        - name: external-drivers
+          persistentVolumeClaim:
+            claimName: pvc-hive-drivers
+      containers:
+        - name: hive-driver
+          image: docker.stackable.tech/stackable/tools:1.0.0-stackable24.3.0
+          env:
+            - name: DEST_DIR
+              value: "/stackable/externals"
+          command:
+            [
+              "bash",
+              "-x",
+              "-o",
+              "pipefail",
+              "-c",
+              "stat ${DEST_DIR}/mysql-connector-j-8.0.31.jar",
+            ]
+          volumeMounts:
+            - name: external-drivers
+              mountPath: /stackable/externals
+----
+
+== Create a Hive Cluster
+
+The MySql connection details can then be used in the definition of the Hive Metastore resource. Note that it is also necessary to "tell" Hive where to find the driver. This is done by setting the value of the environment variable `METASTORE_AUX_JARS_PATH` to the path of the mounted driver:
+
+[source,yaml]
+----
+---
+apiVersion: hive.stackable.tech/v1alpha1
+kind: HiveCluster
+metadata:
+  name: hive-with-drivers
+spec:
+  image:
+    productVersion: 3.1.3
+  clusterConfig:
+    database:
+      connString: jdbc:mysql://mysql:3306/hive  # <1>
+      user: hive  # <2>
+      password: hive
+      dbType: mysql
+    s3:
+      reference: minio  # <3>
+  metastore:
+    roleGroups:
+      default:
+        envOverrides:
+          METASTORE_AUX_JARS_PATH: "/stackable/external-drivers/mysql-connector-j-8.0.31.jar"   # <4>
```

Comment:
Did not test this, would prefer this if multiple drivers are added?
```suggestion
          METASTORE_AUX_JARS_PATH: "/stackable/external-drivers/*"   # <4>
```

Reply:
I saw some examples with the comma seperated list, but people said pointing e.g. to hdfs://.../* is fine. So i would assume it works but fine with leaving it as is.